src/main/java/io/reactivex/Single.java

```diff
@@ -345,7 +345,7 @@
     }
 
     /**
-     * Concatenates a Publisher sequence of Publishers eagerly into a single stream of values.
+     * Concatenates a Publisher sequence of Single sources eagerly into a single stream of values.
```

Comment:
`SingleSource`s

Reply:
hm.. i can see this "Single sources" instead of SingleSources in a few other places within the Javadoc of the Single class - guess we should fix all?

Reply:
Sure. I think they are remnants from the time when there was no `SingleSource` interface yet.

Reply:
oh i see - fixed in all places within the Single class.